main.nf

```diff
@@ -3,76 +3,118 @@
 ========================================================================================
                          nf-core/mag
 ========================================================================================
- nf-core/mag Analysis Pipeline. Started 2018-05-22.
- #### Homepage / Documentation
- https://github.com/nf-core/mag
- #### Authors
- Hadrien Gourlé HadrienG <hadrien.gourle@slu.se> - hadriengourle.com>
+
+nf-core/mag Analysis Pipeline. Started 2018-05-22.
+#### Homepage / Documentation
+https://github.com/nf-core/mag
+#### Authors
+Hadrien Gourlé HadrienG <hadrien.gourle@slu.se> - hadriengourle.com>
+Daniel Straub <d4straub@gmail.com>
 ----------------------------------------------------------------------------------------
 */
 
 
 def helpMessage() {
+    log.info nfcoreHeader()
     log.info"""
-    =========================================
-     nf-core/mag v${params.version}
-    =========================================
     Usage:
 
     The typical command for running the pipeline is as follows:
 
     nextflow run nf-core/mag --reads '*_R{1,2}.fastq.gz' -profile docker
+    nextflow run nf-core/mag --manifest 'manifest.tsv' -profile docker
 
     Mandatory arguments:
       --reads                       Path to input data (must be surrounded with quotes)
```

Comment:
Maybe wanna clarify if this is for short reads, long reads or both?

Reply:
It seems to be using `--readPaths` instead?

Reply:
I think we got that covered with #25